fiftyone/operators/types.py

```diff
@@ -1865,3 +1865,27 @@ def __init__(self, **kwargs):
         if placement not in ["left", "right"]:
             raise ValueError('placement must be either "left" or "right".')
         super().__init__(**kwargs)
+
+
+class IconButtonView(Button):
+    """Represents a button in a :class:`View`.
+
+    Examples::
+
+        import fiftyone.operators.types as types
+
+        iconButtonView = types.IconButtonView(
+            icon="waving_hand",
+            operator="print_stdout",
+            params={"msg": "Hi!"},
+        )
+
+        inputs = types.Object()
+        inputs.view("icon_btn", iconButtonView)
```

Comment:
We should add `inputs.btn(name, icon="waving_hand", variant="round")` since this is going to be such a typical use case. I'd like to keep first class usecases using the `types.Object.util()` methods (eg. `.str()`, `.enum()`).